cudl-sandbox/terraform.tfvars

```diff
@@ -87,7 +97,35 @@ source-bucket-sqs-notifications = [
     "queue_name"    = "CUDLIndexQueue"
     "filter_prefix" = "solr-json/"
     "filter_suffix" = ".json"
-    "bucket_name"   = "cudl-dist"
+    "bucket_name"   = "cudl-data-releases"
+  },
+#   {
+#     "type"          = "SQS",
+#     "queue_name"    = "CUDLIndexCollectionQueue"
+#     "filter_prefix" = "collection/"
+#     "filter_suffix" = ".json"
+#     "bucket_name"   = "cudl-data-releases"
+#   },
+#   {
+#     "type"          = "SQS",
+#     "queue_name"    = "CUDLPackageDataUpdateDBQueue"
+#     "filter_prefix" = "collections/"
+#     "filter_suffix" = ".json"
+#     "bucket_name"   = "cudl-data-releases"
+#   },
```

Comment:
This section could be removed as it will still exist in git history

Reply:
Removed.